packages/sources/svelte/src/components/structure/VtmnListItem/VtmnListItem.svelte

```diff
@@ -71,12 +75,20 @@
   const checkSlotExists = (slotName) => {
     return SLOTS && SLOTS[slotName] && SLOTS[slotName].length;
   };
+
+  const handleSelectItem = (e) => {
+    if (!e.keyCode || e.keyCode === ENTER) {
+      dispatch('click');
+    }
+  };
 </script>
 
 <!-- svelte-ignore a11y-role-has-required-aria-props -->
 <!-- This will be refactored in next major release -->
 <li
-  on:click
+  bind:this={listElement}
+  on:click|stopPropagation={handleSelectItem}
+  on:keydown|stopPropagation={handleSelectItem}
   class={componentClass}
   role="option"
```

Comment:
If element is clickable maybe a `role="button"` is more appropriate

Reply:
yes, by default the css will displayed an "option" rôle. 
Should we change it on button for every `VtmnListItem` ?

Reply:
I guess for screen reader it will be clearer if it's identified like a button. You should try because i'm not sure.